torch/_inductor/codegen/triton.py

```diff
@@ -1569,7 +1571,7 @@ def split_and_set_ranges(self, lengths: List[List[sympy.Expr]]):
 
     def is_indirect_indexing(self, index: sympy.Expr):
```

Comment:
Do we want to reuse this function everywhere? It'd be quite nice.

Reply:
I'm not... opposed? But it also doesn't seem like that much benefit haha

Reply:
It's just easier to understand what's going on. `TMP` has never been a particularly descriptive name lol